activesupport/lib/active_support/railtie.rb

```diff
@@ -124,6 +124,8 @@ class Railtie < Rails::Railtie # :nodoc:
           ActiveSupport.deprecator.warn("config.active_support.remove_deprecated_time_with_zone_name is deprecated and will be removed in Rails 7.2.")
         elsif k == "use_rfc4122_namespaced_uuids"
           ActiveSupport.deprecator.warn("config.active_support.use_rfc4122_namespaced_uuids is deprecated and will be removed in Rails 7.2.")
+        elsif k == :assertionless_tests_behavior
```

Comment:
It looks like we need an explicit case if we want to do something else other than simple assignment. But for `assertionless_tests_behavior` the `else` branch should handle it.

Reply:
`else` branch assigns to `ActiveSupport`, but we need to assign to `ActiveSupport::TestCase`. I can move it to the `else` branch, but not sure it will be better.

Reply:
Missed that 🙈 Thanks for clarifying!